### DISCUSSION TITLE
Esiste un Sonoff con ingressi per rilevare la Presenza Tensione 230 V?

### DISCUSSION THREAD

**Discussion by vespadj** [indomus/forum#1018] [Hardware & Firmware]
Esiste un Sonoff o simile con ingressi per rilevare la Presenza Tensione 230 V?

Oppure un progettino semplice diy con ESP32?

Mi occorrono 5 ingressi e 4 uscite 230 V

**Comment by pin010** [comment]
Un piccolo contributo, un front-end con soli 4 ingressi...
![image](https://user-images.githubusercontent.com/81494843/141204441-9f802d13-d51f-4aac-a5f1-7e4550048df3.png)

**Comment by vespadj** [comment]
Mi sto orientando per mettere un relè per ogni ingresso.
Bobina: 230V - Neutro; contatti verso ESP32: 3V - GPIOx

**Reply by pin010** [reply]
Perchè no :-)

**Comment by pin010** [comment]
I don't know your projects but also take a look at  Shelly i3, it could be a nice shortcut
